scripts/run_examples.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Script should be run from root directory(zsh scipts/run_examples.sh PathToExamples AccountID PrivateKey)
+# Keep in mind this is created to correctly run executables which accept arguments [AccountID, PrivateKey]
+# Extract command-line arguments
+exe_dir="$1"
+arg1="$2"
+arg2="$3"
+
+# Check if the executable directory exists
+if [ ! -d "$exe_dir" ]; then
+    echo "Error: The specified directory does not exist."
+    exit 1
+fi
+
+# Iterate through the executables in the directory
+for exe in "$exe_dir"/*; do
+    if [ -x "$exe" ]; then
+        echo "Running $exe with arguments: $arg1 $arg2"
+        "$exe" "$arg1" "$arg2"
+        echo "Finished running $exe"
+    fi
+done
```

Comment:
Is there a way to add being able to just run executable? Some of the examples never finish (end with a `while (true) {}` loop) so this won't run through all examples anyways

Reply:
I might recommend installing this script as well, so anybody who downloads a release can run examples the same way

Reply:
Also, not all examples run with the same two input arguments (see `ConstructClientExample` and `GeneratePrivateKeyFromMnemonicExample`). Should these maybe be included in a separate script? Or possibly a script made for each example?

Reply:
Will move the script topic to a separate discussion and issue.